setup.py

```diff
@@ -29,11 +29,26 @@
 from horovod import __version__
 
 
+class CMakeExtension(Extension):
+    def __init__(self, name, cmake_lists_dir='.', sources=[], **kwa):
+        Extension.__init__(self, name, sources=sources, **kwa)
+        self.cmake_lists_dir = os.path.abspath(cmake_lists_dir)
+
+
 tensorflow_mpi_lib = Extension('horovod.tensorflow.mpi_lib', [])
 torch_mpi_lib = Extension('horovod.torch.mpi_lib', [])
 torch_mpi_lib_impl = Extension('horovod.torch.mpi_lib_impl', [])
 torch_mpi_lib_v2 = Extension('horovod.torch.mpi_lib_v2', [])
 mxnet_mpi_lib = Extension('horovod.mxnet.mpi_lib', [])
+gloo_lib = CMakeExtension('gloo', cmake_lists_dir='third_party/gloo', sources=[])
+
+
+def find_files(base_dir, ext):
```

Comment:
Do we use this function?

Reply:
It's legacy code, but I think we do use this function.

Reply:
A typo. We do not use the function. I think it's fine to remove it.